src/Components/Publishing/Header2/Layouts/Components/FeatureFullscreenHeader.tsx

```diff
@@ -0,0 +1,115 @@
+import { space } from "@artsy/palette"
+import { PartnerInline } from "Components/Publishing/Partner/PartnerInline"
+import React from "react"
+import styled from "styled-components"
+import { resize } from "../../../../../Utils/resizer"
+import { pMedia } from "../../../../Helpers"
+import { EditImage, FeatureHeaderProps } from "../FeatureHeader"
+import { FeatureInnerContent } from "./FeatureInnerContent"
+
+export const FeatureFullscreenHeader: React.SFC<FeatureHeaderProps> = props => {
+  const {
+    article: { hero_section, is_super_article, seriesArticle, super_article },
+    editImage,
+  } = props
+  const url = (hero_section && hero_section.url) || ""
+  const isVideo = url.includes("mp4")
+  const src = !isVideo && url.length && resize(url, { width: 1600 })
+
+  return (
+    <FeatureHeaderContainer hasNav={seriesArticle && true}>
+      <FeatureAssetContainer src={src ? src : undefined}>
```

Comment:
why not just `src={src}`?

Reply:
This prop determines the background image, so undefined prevents a background image from being applied, otherwise it gets `background-image: url('')`